index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module "norway-postal-codes" {
+  const postalCodes: Record<string, string>;
+
+  export default postalCodes;
+}
```

Comment:
Please follow this style guide: https://github.com/sindresorhus/typescript-definition-style-guide/tree/57a925bb5b3eb46a4666eab31e1c883a26b7bfb5

Reply:
I think it should be compliant with the style guide now